app/models/suggestion.py

```diff
@@ -1,22 +1,27 @@
 from datetime import datetime
 import pytz
+
 from .. import db
+from ..models.resource import Resource, ResourceBase
+
 
-class Suggestion(db.Model):
+class ResourceSuggestion(ResourceBase):
     """
     Association between a resource and potential suggestions for it
     """
-    __tablename__ = 'suggestions'
-    id = db.Column(db.Integer, primary_key=True)
-    resource_id = db.Column(db.Integer, db.ForeignKey('resources.id'))
-    suggestion_text = db.Column(db.String(250))
```

Comment:
I can update the suggestion tests in a future PR. They didn't successfully pass with the old models either, I believe.